intercom-java/src/main/java/io/intercom/api/Event.java

```diff
@@ -94,6 +95,9 @@ static void validateCreateEvent(Event event) {
     @JsonProperty("email")
     private String email;
 
+    @JsonProperty("id")
+    private String id;
```

Comment:
Could the field be renamed to `intercomUserId`? I think it would be more clear as `id` in this case could be understood as event id.